Package.swift

```diff
@@ -8,7 +8,32 @@ func resolveTargets() -> [Target] {
     let baseTargets: [Target] = [
         .target(name: "RevenueCat",
                 path: ".",
-                exclude: ["Purchases/Info.plist"],
```

Comment:
Apparently Swift 5.5's SPM will try to parse everything that isn't explicitly excluded, and that causes Xcode to take forever and raise warnings if you add the package. 
Excluding all paths manually solves it. Would love ideas, though. 
I think ultimately we just need to clean up the file structure for the repo, it's really messy right now.